src/main/java/baseball/result/Result.java

```diff
@@ -0,0 +1,54 @@
+package baseball.result;
+
+import baseball.number.GameNumber;
+
+public class Result {
+    private int strikeCount;
+    private int ballCount;
+    public Result() {
+        this.strikeCount = 0;
+        this.ballCount = 0;
+    }
+
+    public void calculateResult(GameNumber seedNumber, GameNumber userNumber) {
+        calculateStrikeCount(seedNumber, userNumber);
+        calculateBallCount(seedNumber, userNumber);
+    }
+
+    private void calculateStrikeCount(GameNumber seedNumber, GameNumber userNumber) {
+        int strikeCount =0;
+        String seedValue = seedNumber.getValue();
+        String userValue = userNumber.getValue();
+
+        for (int index = 0; index < seedValue.length(); index++) {
```

Comment:
for 문의 i,j,k의 경우 약어라고 생각하지는 않는것 같더라구요 🙂
물론 i가 index보다 낫다 라는 뜻은 아니지만, 어느쪽을 사용하던 상관 없을것 같아요. 
(i, j, k 대신 a, b, foo, bar 와 같은 의미없는 문자를 쓰지 말라는 글은 본것 같아요)
```suggestion
        for (int i = 0; i < seedValue.length(); i++) {
```